backend/tests/test_contact_type.py

```diff
@@ -37,3 +37,62 @@ def test_contact_type_routes(self):
         # deleting a contact_type that does not exist
         response = self.client.delete("/contact_type/{}".format(ct_id))
         self.assertEqual(response.status_code, 404)
+        
+    def test_create_contact_type_route(self):
+        # creating a contact_type with valid arguments
+        valid_response = self.client.post(
+            "/contact_type",
+            json={
+                "hex_colour": "#F0F8FF",
+                "type": "dummy type",
+                "description": "dummy description",
+            },
+        )
+
+        # creating a contact_type with valid arguments
+        valid_response2 = self.client.post(
+            "/contact_type",
+            json={
+                "hex_colour": "#ffffff",
+                "type": "new dummy type",
+                "description": "new dummy description",
+            },
+        )
+
+        # creating a contact_type with valid arguments
+        valid_response3 = self.client.post(
+            "/contact_type",
+            json={
+                "hex_colour": "#ffffff",
+                "type": "dummy type 3",
+                "description": "dummy description 3",
+            },
+        )
+
+        # creating a contact_type with empty arguments
+        empty_response = self.client.post(
+            "/contact_type",
+            json={
+                "hex_colour": "",
+                "type": "",
+                "description": "",
+            },
+        )
+
+        # checking that the tests worked
+        self.assertEqual(valid_response.status_code, 200)
+        self.assertEqual(valid_response2.status_code, 200)
+        self.assertEqual(valid_response3.status_code, 200)
+        self.assertEqual(empty_response.status_code, 400)
+
+        returned_json = self.client.get("/contact_type")
+        json_data = returned_json.get_json()
```

Comment:
@carolaltimas As the route doesn't exist, it will return `None`, which is not a valid argument to `len()`. You can try testing your creation by verifying the json returned from your creation route.